.github/workflows/sign.yml

```diff
@@ -24,111 +24,32 @@ jobs:
   Sign:
     runs-on: windows-latest
     steps:
+    - name: Checkout repository
```

Comment:
Why do we need this?

Reply:
Anytime you're referencing files in the repo from within a workflow, you need to checkout the repo. Since we're referencing the `sign.ps1` script, we need to checkout.